From d9af8a2d14791829052e969e459c53622a07117f Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Mon, 16 Oct 2023 12:09:16 +0200
Subject: [PATCH 0006/1695] core: arm: add guards to sm/psci.h

Adds missing header guards in sm/psci.h.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 339980d460dbeb831671702e83eba07c05a70d1a core: arm: add guards to sm/psci.h]
Change-Id: I0a4b35db735fa69f094d50ee68989fd2e5851005
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
[etienne: cherry-picked from 62d8e0e8447138133f8e09bb7e52dba0d6dabf6b]
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
---
 core/arch/arm/include/sm/psci.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/arch/arm/include/sm/psci.h b/core/arch/arm/include/sm/psci.h
index da873615c..95ccdfa4c 100644
--- a/core/arch/arm/include/sm/psci.h
+++ b/core/arch/arm/include/sm/psci.h
@@ -2,6 +2,9 @@
 /*
  * Copyright (c) 2017-2023, Linaro Limited
  */
+#ifndef __SM_PSCI_H
+#define __SM_PSCI_H
+
 #include <kernel/thread.h>
 #include <sm/sm.h>
 #include <stdint.h>
@@ -83,3 +86,4 @@ int psci_stat_count(uint32_t cpu_id, uint32_t power_state);
 void tee_psci_handler(struct thread_smc_args *args, struct sm_nsec_ctx *nsec);
 
 void psci_armv7_cpu_off(void);
+#endif /* __SM_PSCI_H */
-- 
2.47.2

